docker-compose.yaml

```diff
@@ -4,8 +4,6 @@ services:
     container_name: "web-server"
     build: ./server
     command: cargo watch -x 'run'
-    depends_on:
-      - postgres
```

Comment:
I was running an outdated version of docker-compose (v1.29.2), so depends_on was doing less than it does with v2. After updating to v2, I removed the depends_on entirely to be able to do some cargo stuff in the background while waiting for the database to be ready. This only has a meaningful effect in cases where the postgres_data volume needs to be rebuilt, and only costs 0.07 seconds on my machine in cases where the postgres_data volume does not need to be rebuilt.